README.md

```diff
@@ -1,237 +1,61 @@
-# Class Extensions and Inheritance
+# Class Extensions and Inheritance: `extends`
 
 ## Learning Goals
 
 - Use the `extends` keyword
-- Use the `super` method
-- Explain when `class` inheritance should be used in JavaScript
 
 ## Introduction
 
 In JavaScript, as in other Object Oriented languages, we've learned
-that you can create `class`es and build methods that can perform
-actions on instance data, or specific to the `class`. What if you have
-`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
+that you can create classes and build methods that can perform
+actions on instance data, or specific to the class. What if you have
+classes that exhibit many of the same behaviors, such as `Cat`, `Dog`,
```

Comment:
I think it may be good to have a code snippet for Cat Dog and Bird classes here to refresh student's memory